src/EA.Weee.Web/Areas/Admin/Controllers/HomeController.cs

```diff
@@ -85,6 +85,9 @@ public ActionResult ChooseActivity(ChooseActivityViewModel viewModel)
                 case InternalUserActivity.ProducerDetails:
                     return RedirectToAction("Search", "Producers");
 
+                case InternalUserActivity.DirectRegistrantDetails:
+                    return RedirectToAction("Search", "Producers");
+
```

Comment:
@stevelee-civica  are there any controller unit tests to update for this?

Reply:
added inline data to HttpPost_ChooseActivity_RedirectsToCorrectControllerAction
to verify RedirectToAction